x-pack/plugins/console_extensions/spec/overrides/ilm.put_lifecycle.json

```diff
@@ -0,0 +1,54 @@
+{
+  "ilm.put_lifecycle": {
+    "data_autocomplete_rules": {
+      "phases": {
+        "warm": {
+          "after": "",
+          "actions": {
+            "allocate": {
+              "number_of_replicas": "",
+              "include": {},
+              "exclude": {},
+              "require": {
+                "_name": ""
+              }
+            },
+            "shrink": {
+              "number_of_shards": ""
+            },
+            "forcemerge": {
+              "max_num_segments": ""
+            }
+          }
+        },
+        "cold": {
+          "after": "",
+          "actions": {
+            "allocate": {
+              "number_of_replicas": "",
+              "include": {},
+              "exclude": {},
+              "require": {
+                "_name": ""
+              }
+            }
+          }
+        },
+        "hot": {
+          "after": "",
+          "actions": {
+            "rollover": {
+              "max_age": ""
```

Comment:
there are more options here? may use a scope link to the rollover api?

Reply:
added options manually, could not get a scope link working properly.

Reply:
interesting. It should work and is worthwhile fixing at some point as it is key to keeping things consistent. I agree it's out of scope for this PR.